pkg/setup/initialize.go

```diff
@@ -138,6 +139,17 @@ func GenerateK8sCerts(ctx context.Context, currentNamespaceClient kubernetes.Int
 		"*." + etcdService + "-headless" + "." + currentNamespace,
 	}
 
+	//expect up to 20 etcd members, number could be lower since more
```

Comment:
this is what shouldn't have been remove from the other pr, extra comment is from there https://etcd.io/docs/v3.3/faq/#what-is-maximum-cluster-size

Reply:
should we go ahead with the current number? I guess its just about expecting at most 20 members and their corresponding SANs and not actually having that many members right?

Reply:
yeah it's just a comment, we can modify it later if we want, I just wanted to point that expecting 20 might be overkill

Reply:
the part that says that it shouldn't have been removed is the code, not the comment btw